core/src/test/java/org/infinispan/stream/BaseStreamIteratorTest.java

```diff
@@ -273,14 +278,17 @@ public Object visitRemoveCommand(InvocationContext ctx, RemoveCommand command) t
          }
       }, 0);
 
-      for (Iterator<Map.Entry<Object, Object>> it = cache(0, CACHE_NAME).getAdvancedCache().withFlags(
-              Flag.SKIP_CACHE_STORE).entrySet().stream().iterator(); it.hasNext();) {
-         Map.Entry<Object, Object> entry = it.next();
-         Object key = entry.getKey();
-         assertEquals(values.get(key), entry.getValue());
-         it.remove();
-      }
-
-      assertEquals(0, cache.size());
+      Iterator<Map.Entry<Object, Object>> it = cache(0, CACHE_NAME).getAdvancedCache().withFlags(
+              Flag.SKIP_CACHE_STORE)
```

Comment:
You can use `advancedCache(0, CACHE_NAME)` and fit it on one line

Reply:
Ah wow I never noticed that method 👍